src/components/Input.tsx

```diff
@@ -177,14 +179,18 @@ const Input: React.FC = () => {
                         : `${displayFormat}${asSingle ? "" : ` ${separator} ${displayFormat}`}`
                 }
                 value={inputText}
+                id={inputId}
+                name={inputName}
+                autoComplete="off"
+                role="presentation"
                 onChange={handleInputChange}
             />
 
             <button
                 type="button"
                 ref={buttonRef}
                 disabled={disabled}
-                className="absolute right-0 h-full px-3 text-gray-400 focus:outline-none disabled:opacity-40 disabled:cursor-not-allowed"
+                className="absolute top-0 right-0 h-full px-3 text-gray-400 focus:outline-none disabled:opacity-40 disabled:cursor-not-allowed"
```

Comment:
Another minor fix I hope you don't mind me including. At least for me, the button was misaligned when rendering in my modal. Setting `top-0` fixed the positioning in all my use-cases.

Reply:
I think it would be much better if we add `toggleClassName` prop to pass in `Datepicker` component, so you can override the toggle button's class.

Reply:
Sure. Do we want to just have it append classes or overwrite them completely?

Reply:
Added `toggleClassName` to the button!